src/Commands/Release/Release.php

```diff
@@ -45,6 +45,18 @@ protected function configureOptions()
                 InputOption::VALUE_NONE,
                 'Skip the text collection task when performing the release'
             )
+            ->addOption(
+                'skip-i18n-tx-pull-and-update',
+                null,
+                InputOption::VALUE_NONE,
+                'Skip running tx pull, i18nTextCollectorTask and js/json updates'
+            )
+            ->addOption(
+                'i18n-tx-push',
+                null,
+                InputOption::VALUE_NONE,
+                'Run tx push -s - only do this during the beta1 release of a new minor'
+            )
```

Comment:
>> What's the expected outcome if you provide both those flags simultaneously?

documented in description:
```
push only (unlikely to ever use this)
cow release:translate -vvv --i18n-tx-push --skip-i18n-tx-pull-and-update --skip-fetch-tags 4.10.0-beta1 silverstripe/installer
```

>> I wouldn't be able to tell what they do without looking a the source code / I would rather the flags describe what's happening rather than the actual operations that are being run

Also I think that `i18n-tx-pull-and-update` is currently more descriptive than --translation-operation:pull, because it also `updates` meaning it runs the text collector and does a bunch of stuff to update the files

Does naming really even matter?  The intention is really that we just copy paste instructions the correct instructions out of confluence doc.  Any idea what this flag does without looking at the source code? `skip-emulate-requirements`.  I have no idea.

Basically the idea of this approach is this:
- without flag, do a pull because it's safe
- if you want to push, put a flag in because it might be unsafe

Reply:
I knew up-front what you were trying to implement and I had to read the thing a few times to figure out what combination of parameters I was meant to use to achieve a given results.

Yes it's an internal tool, but we should aim to make it easy for us to understand what it's doing both to minimise the risk of error and to make it easier to onboard new devs.

Seems to me like it's worth spending 5 min to make this clear and concise and save us pointless doc reading on every follow up release.

Reply:
I changed `tx` to `translation`